octoprint_enclosure/templates/enclosure_settings.jinja2

```diff
@@ -808,7 +808,7 @@
           <option value="high">High</option>
           <option value="toggle">Toggle</option>
         </select>
-        <span class="help-inline">When the event happen, you want to turn the controlled IO HIGH or LOW?</span>
+        <span class="help-inline">When the event happens, you do want to turn the controlled IO HIGH or LOW?</span>

```

Comment:
Should be "do you" instead of "you do"

Reply:
I'm new to GitLab so I'm not quite sure what you need.  Do I need to make a new PR for this?